src/main/java/at/ac/fhcampuswien/fhmdb/HomeController.java

```diff
@@ -13,104 +13,160 @@
 import javafx.collections.ObservableList;
 import javafx.collections.transformation.FilteredList;
 import javafx.fxml.FXML;
-import javafx.fxml.Initializable;
 import javafx.scene.control.TextField;
 
+import java.io.IOException;
 import java.net.URL;
 import java.util.*;
-import java.util.stream.Collectors;
-import java.util.stream.IntStream;
 
-public class HomeController implements Initializable {
+public class HomeController {
```

Comment:
Why did you remove the `Initializable` interface?

Reply:
So that the initialize method could be tested without the need to pass the URL and resources parameters. Now they are auto-injected with @FXML annotations as class properties.